src/Components/AddButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {View, TouchableOpacity, Text, StyleSheet} from 'react-native';
+
+type props = {
+  setShowCrypto: (value: boolean) => void;
+};
+
+export default function AddButton({setShowCrypto}: props) {
+  return (
+    <View>
+      <TouchableOpacity
+        style={styles.buttonContainer}
+        onPress={() => setShowCrypto(true)}>
+        <Text style={styles.buttonAdd}>+ Add a Cryptocurrency</Text>
+      </TouchableOpacity>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
```

Comment:
styles should be stored in a separate file, the most common way to do this is with this format:

src/
   Components/
      AddButton/
          index.tsx <---- this file 
          styles.ts <---- move the styles here